lfs/config.go

```diff
@@ -544,6 +565,38 @@ func (c *Configuration) readGitConfig(output string, uniqRemotes map[string]bool
 	}
 }
 
+// Special handling for url.<url>.insteadOf, since <url> will contain '.'
+func (c *Configuration) readGitInsteadOfConfig(output string, uniqInsteadOfs map[string]string) {
+	lines := strings.Split(output, "\n")
+	uniqKeys := make(map[string]string)
+	for _, line := range lines {
+		// Why is this not "="?
+		pieces := strings.SplitN(line, " ", 2)
```

Comment:
Doh, sorry. That's silly of me. No clue why `git config --list` differs from `--get-regexp`, either. Looks like the only way we could parse the output of both commands the same is by using `-z`.